samples/nrf9160/fmfu_smp_svr/prj.conf

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# Enable mcumgr.
+CONFIG_MCUMGR=y
+
+# Some command handlers require a large stack.
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+# Ensure an MCUboot-compatible binary is generated.
+CONFIG_BOOTLOADER_MCUBOOT=y
+
+# Enable the UART mcumgr transports.
+CONFIG_MCUMGR_SMP_UART=y
+CONFIG_UART_MCUMGR_RX_BUF_SIZE=512
+CONFIG_MCUMGR_BUF_SIZE=512
+
+# Required by the `taskstat` command.
+CONFIG_THREAD_MONITOR=y
+
+# Enable statistics and statistic names.
+CONFIG_STATS=y
+CONFIG_STATS_NAMES=y
+
+# Enable STAT and OS from the core commands
+CONFIG_MGMT_FMFU=y
+CONFIG_MCUMGR_CMD_OS_MGMT=y
+CONFIG_MCUMGR_CMD_STAT_MGMT=y
+
+CONFIG_LIBMODEM=y
+CONFIG_LIBMODEM_SYS_INIT=y
+CONFIG_NRF_MODEM_BUILD_FROM_SRC=y
+CONFIG_AT_CMD=y
+CONFIG_AT_CMD_PARSER=y
+CONFIG_AT_CMD_SYS_INIT=y
+CONFIG_MODEM_INFO=y
+
+CONFIG_NETWORKING=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_OFFLOAD=y 
+CONFIG_HEAP_MEM_POOL_SIZE=8192
+CONFIG_SPEED_OPTIMIZATIONS=y
+
+# RTT Logging since you cannot use the UART as it is used for SMP
```

Comment:
It is possible, however, only the one sets of pins are connected to the Segger chip AFAIK so you don't have 2x virtual serial ports for the APP core.

The 3 ports which are enumerated for the nrf9160 board are 1x nrf9160 app core, 1x nrf9160 modem, and 1x nrf52840(board controller). There is a UART line between the nrf9160 app and nrf52840 however this would require the sample to set up the nrf52840 with custom board controller firmware.

Reply:
Ah, so we're limiting default sample configurations to what is available on the DKs out of the box? In this case, virtual COM ports.

I guess my curiosity is about Zephyr driver conflicts between serial logging and SMP. For example, plugging an FTDI with some jumpers into pins enabled by APP for one of its UARTE instances, then getting output that way rather than RTT.

Reply:
That's totally possible. No conflicts you just change the output instance